src/main/com/fulcrologic/fulcro/routing/dynamic_routing.cljc

```diff
@@ -314,18 +314,38 @@
         root       (ast-node-for-live-router app ast)
         to-signal  (atom [])
         to-cancel  (atom [])
-        _          (loop [{:keys [component] :as node} root new-path-remaining new-path]
+;; TODO: Remove next line
+;;        _          (loop [{:keys [component] :as node} root new-path-remaining new-path]
+        _          (loop [{:keys [component children] :as node} root new-path-remaining new-path]
                      (when (and component (router? component))
                        (let [new-target    (first new-path-remaining)
                              router-ident  (comp/get-ident component {})
                              active-target (get-in state-map (conj router-ident ::current-route))
                              {:keys [target]} (get-in state-map (conj router-ident ::pending-route))
-                             next-router   (some #(ast-node-for-live-router app %) (:children node))]
+;; TODO: Remove next line
+;;                             next-router   (some #(ast-node-for-live-router app %) (:children node))]
+;; FIXME: Resolve reference to reconciler => /portfolio/fulcro2/src/main/fulcro/client/primitives.cljc
+                               next-router   (some #(ast-node-for-live-router reconciler %) children)]
                          (when (eql/ident? target)
                            (swap! to-cancel conj target))
                          (when (and (not= new-target active-target) (vector? active-target))
-                           (when-let [c (comp/ident->any app active-target)]
-                             (swap! to-signal conj c)))
+;; TODO: Remove next lines
+;;                           (when-let [c (comp/ident->any app active-target)]
+;;                             (swap! to-signal conj c)))
+;;;;;;;;
+;; FIXME: Resolve references to prim namespace => /portfolio/fulcro2/src/main/fulcro/client/primitives.cljc
+                              (let [mounted-target-class (reduce (fn [acc {:keys [dispatch-key component]}]
+                                                                   (when (= ::current-route dispatch-key)
+                                                                     (reduced component)))
+                                                           nil
+                                                           (some-> component (prim/get-query state-map)
+                                                             prim/query->ast :children))
+                                    mounted-targets      (prim/class->all reconciler mounted-target-class)]
```

Comment:
Porting `prim/class->all` to it's `comp/class->all` counterpart.

Reply:
yes